lua/refactoring/utils.lua

```diff
@@ -10,8 +10,8 @@ M.get_root = function(bufnr, lang)
 end
```

Comment:
Did I not run stylua on this or did you run something other than stylua?

Reply:
i did run luafmt thats why i reverted the change